cmd/tsidp/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
we already have this at the top level of the repo. Let's not copy it to every directory?